tests/lib/naming_test.py

```diff
@@ -169,5 +171,123 @@ def testParseServiceFile(self):
     filedefs._ParseFile(data, 'services')
     self.assertEqual(filedefs.GetService('HTTP'), ['80/tcp'])
 
+
+class Naming_Characterization_Tests(unittest.TestCase):
```

Comment:
A lot of these look to be exact duplicates of the unittests. Does naming.py really need characterization tests?